packages/medusa/src/services/__tests__/cart.js

```diff
@@ -515,6 +515,21 @@ describe("CartService", () => {
       expect(LineItemAdjustmentServiceMock.delete).toHaveBeenCalledTimes(1)
       expect(LineItemAdjustmentServiceMock.delete).toHaveBeenCalledWith({
         item_id: [IdMap.getId("merger")],
+        discount_id: expect.objectContaining({
```

Comment:
suggestion: would it work to pass `expect.objectContainer(Not(IsNull))`?

Reply:
yep, it's more elegant 👍